cli/src/generate/templates/gitignore

```diff
@@ -0,0 +1,18 @@
+# Rust artifacts
+/Cargo.lock
+/target
+
+# Node artifacts
+/build
+/node_modules
+/package-lock.json
+
+# Zig artifacts
+/zig-cache/
+/zig-out/
+
+# Example dirs
+/examples/*/
+
+# Grammar extras
+/dsl.d.ts
```

Comment:
Swift will produce a `.build` directory at the top level. Including a `\.build` ignore rule can catch that one.